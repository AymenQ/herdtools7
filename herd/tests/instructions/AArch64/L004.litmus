AArch64 CSNEG

{
uint64_t 0:X0=1;
uint64_t 0:X2=2;
uint64_t 0:X1;
uint64_t 0:X3;
}
 P0                 ;
 CMP X0,X0          ;
 CSNEG X1,X0,X2,EQ  ;
 CSNEG X3,X0,X2,NE  ;
locations [0:X1; 0:X3;]
