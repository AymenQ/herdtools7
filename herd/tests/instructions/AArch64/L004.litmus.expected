Test CSNEG Required
States 1
0:X1=1; 0:X3=-2;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (true)
Observation CSNEG Always 1 0
Hash=fc337307eaf34d1bdef16673a0ee4f80

